### DISCUSSION TITLE
Fake GC-01: Scam?

### DISCUSSION THREAD

**Discussion by micha-blip** [Gissio/radpro#154] [Q&A] [answered]
Hi, I bought this device and I wonder what it is actually... 

is it even a functional device or did I get scammed?

![IMG_4764](https://github.com/user-attachments/assets/6861d802-9cf8-4bd2-b50f-e825103f47a0)

Thanks!

**Comment by jonwhite227** [comment, marked as answer]
It's definitely not a genuine GC-01, but one of the cheap copies. Does it turn on? If it works, it won't be very sensitive as the J-613/614 tubes are almost useless. It shouldn't have an 18650 or similar cell stuck to the board either. 
Send it back if you can and get one from Fnirsi Official store on AliExpress, though they might supply with the cheap tube, but at least you can swap it out for a J321, install RadPro and have a functional unit.

**Comment by micha-blip** [comment]
It does turn on and measures something, but the measurements jump a lot (especially when shaken a bit). 
Thank you for your help!

**Reply by Gissio** [reply]
Let me refer you to https://github.com/Gissio/radpro/issues/103.

Also: J613/J614 tubes are useful, they are just less sensible than their J321/M4011 counterparts.
